pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -558,10 +572,39 @@ func getDashbuilderTemplate(cr *api.KieApp, serversConfig []api.ServerTemplate,
 			dashbuilderTemplate.Cors = *cr.Status.Applied.Objects.Dashbuilder.Cors
 		}
 
+		// Enable EdgeTermination
+		if cr.Status.Applied.Objects.Dashbuilder.TerminationRoute != nil && len(cr.Status.Applied.Objects.Dashbuilder.TerminationRoute.Protocol) > 0 {
+			dashbuilderTemplate.TerminationRoute.Termination, dashbuilderTemplate.TerminationRoute.Protocol = getDashbuilderTermination(cr)
+			dashbuilderTemplate.TerminationRoute.EnableTlsSection = true
```

Comment:
?

Reply:
why is it hard coded `dashbuilderTemplate.TerminationRoute.EnableTlsSection = true` ?

Reply:
Here is set to true because here we need to enable this : https://github.com/kiegroup/kie-cloud-operator/pull/738/files#diff-4c2e80347eb77843cf21753ad56fbeb8dd6fec9b481a9e6bf45a297e279b0f55R406  here we have a different values https://github.com/kiegroup/kie-cloud-operator/pull/738/files#diff-29ad51b906bc49dd54db094a3b626e8e4a04628d03562e6c0d45db0e391ac2fbR583-R586

Reply:
why it is needed to be enabled?

Reply:
Becuse it enable the edge and if not used it replace disableSSL we use to show/hidden fields on the yaml